C/other/Tower_Of_Hanoi.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
+
+void TowerOfHanoi(int n,int a,int b,int c){
+    if(n==1){
+        printf("%d -> %d\n", a,b);
+        return ;
+    }
+    TowerOfHanoi(n-1,a,c,b);
+    printf("%d -> %d\n", a,b);
+    TowerOfHanoi(n-1,c,b,a);
+}
+
+int main(){
+    int n;
```

Comment:
Change the variable name n to something good, like number_of_discs or something like that.
It's necessary for a code to speak itself.